TFT/src/User/API/Language/language_keywords.h

```diff
@@ -66,6 +66,7 @@ extern "C" {
 #define LANG_KEY_STEALTH_CHOP                 "label_stealth_chop:"
 #define LANG_KEY_DELTA_CONFIGURATION          "label_delta_configuration:"
 #define LANG_KEY_DELTA_TOWER_ANGLE            "label_delta_tower_angle:"
+#define LANG_KEY_DELTA_DIAGONAL_ROD            "label_delta_diagonal_rod:"
```

Comment:
remove a space here. IMHO, we should avoid to add other params. This kind of code takes a lot of FLASH (labels, code), it cannot be optimized and it's not even used by the majority of users.

Reply:
Fixed. This params is used by all Delta printers users.